modules/@apostrophecms/piece-type/ui/apos/components/AposDocsManager.vue

```diff
@@ -405,11 +405,11 @@ export default {
     },
     shortcutNew(event) {
       const interesting = event.keyCode === 78; // N(ew)
-      const topModal = apos.modal.stack[apos.modal.stack.length - 1] ? apos.modal.stack[apos.modal.stack.length - 1].id : null;
+      const topModalId = apos.modal.stack.at(-1)?.id;
```

Comment:
Must start using this I see it's been in node and supported browsers for a while